pkg/webhook/create.go

```diff
@@ -48,9 +50,44 @@ func SetupWebhooks(ctx context.Context, cfg *rest.Config) error {
 	if err != nil {
 		return fmt.Errorf("failed to create new client: %w", err)
 	}
+
+	err = setupCluster(ctx, client, namespace)
```

Comment:
I don't get why SA, clusterRole and clusterRole binding is hidden in that setupCluster method while they all are steps of setting up webhook server. I would move everything on the same level or regroup it in the following way: initCfg + RBAC + SecureService + Deployment

Reply:
It was done this way so we don't have 200+ line functions like what Che operator has. It's been changed to initCfg + RBAC + SecureService + Deployment.